src/main/java/com/cburch/logisim/std/gates/Buffer.java

```diff
@@ -259,7 +259,7 @@ public void paintInstance(InstancePainter painter) {
   public void propagate(InstanceState state) {
     var in = state.getPortValue(1);
     in = Buffer.repair(state, in);
-    state.setPort(0, in, GateAttributes.DELAY);
+    state.setPort(0, in.not().not(), GateAttributes.DELAY);
```

Comment:
Please put your comment from the conversation here to mark `not().not()` is for a reason.

Reply:
I agree with @MarcinOrlowski here that a comment like below would be helpful to prevent someone to introduce inadvertantly a bug by removing the double negation.